fabtests/test_configs/tcp/io_uring.exclude

```diff
@@ -70,13 +70,9 @@ fi_msg_sockets
 # fi_unexpected_msg -e rdm fails with no message
 fi_unexpected_msg -e rdm
 
-# fi_bw -e msg fails with
-# fi_eq_sread(): common/shared.c:1165, ret=-4 (Interrupted system call)
-fi_bw -e msg
-
-# fi_bw fails by hanging
+# fi_flood fails by  runfabtest timeout only on  the CI.
```

Comment:
extra spaces here. I'm also tempted to keep the two separate excludes since they are for different reasons so it's better documented separately